specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/2016-06-01/vaults.json

```diff
@@ -50,7 +50,7 @@
         }
       }
     },
-    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/operations": {
+    "/providers/Microsoft.RecoveryServices/operations": {

```

Comment:
@veronicagg - This fix is fine since otherwise the API had no meaning and clients couldnt anyway meaningfully use it. And as @sonathan pointed out, the CLI is not yet released so from that perspective it should be ok as well.